bin/ut-run.js

```diff
@@ -0,0 +1,46 @@
+const pkgJson = require('../package.json');
```

Comment:
The first row must be #!/usr/bin/env node
Also make sure the file has executable flag.
```sh
git update-index --chmod=+x bin/ut-run.js
```

Reply:
[done](https://github.com/softwaregroup-bg/ut-run/pull/100/commits/ff1dbfb32c4a3b6299cbaea18b11209b82682250)